scripts/us_epa/air_emissions_inventory/common/us_air_pollution_emission_trends.py

```diff
@@ -0,0 +1,183 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This Python Script Load the datasets, cleans it
+and generates cleaned CSV, MCF, TMCF file.
+"""
+
+import os
+import sys
+import pandas as pd
+
+sys.path.insert(1, os.path.dirname(os.path.abspath(__file__)))
+
+_SOURCE_GROUPS = {
+    'FuelCombustionElectricUtility': 'StationaryFuelCombustion',
+    'FuelCombustionIndustrial': 'StationaryFuelCombustion',
+    'EPA_FuelCombustionOther': 'StationaryFuelCombustion',
+    'ChemicalAndAlliedProductManufacturing': 'IndustrialAndOtherProcesses',
+    'MetalsProcessing': 'IndustrialAndOtherProcesses',
+    'PetroleumAndRelatedIndustries': 'IndustrialAndOtherProcesses',
+    'EPA_OtherIndustrialProcesses': 'IndustrialAndOtherProcesses',
+    'SolventUtilization': 'IndustrialAndOtherProcesses',
+    'StorageAndTransport': 'IndustrialAndOtherProcesses',
+    'WasteDisposalAndRecycling': 'IndustrialAndOtherProcesses',
+    'OnRoadVehicles': 'Transportation',
+    'NonRoadEnginesAndVehicles': 'Transportation'
+}
+
+
+class USAirPollutionEmissionTrends:
+    """
+    This Class has requried methods to generate Cleaned CSV,
+    MCF and TMCF Files.
+    """
+
+    _mcf_template = ''
+    _tmcf_template = ''
+
+    def __init__(self,
+                 input_files: list,
+                 csv_file_path: str = None,
+                 mcf_file_path: str = None,
+                 tmcf_file_path: str = None) -> None:
+        self._input_files = input_files
+        self._cleaned_csv_file_path = csv_file_path
+        self._mcf_file_path = mcf_file_path
+        self._tmcf_file_path = tmcf_file_path
+        self._final_df = pd.DataFrame(columns=[
+            'geo_Id', 'year', 'SV_TEMP', 'observation', 'Measurement_Method'
+        ])
+
+    def set_cleansed_csv_file_path(self, cleansed_csv_file_path: str) -> None:
```

Comment:
nit: set_cleaned_csv_file_path & cleaned_csv_file_path?

Reply:
changed to set_cleaned_csv_file_path & cleaned_csv_file_path